### ISSUE TITLE
TreeView should mark all non-selected nodes with aria-selected false attribute

### ISSUE THREAD

**Issue by Iankodj** [telerik/kendo-ui-core#3200]
### Bug report

Based on the specification (https://www.w3.org/TR/wai-aria/states_and_properties#aria-selected), no attribute marks the DOM element as not selectable. In order to mark it as selectable, but no selected, the aria-selected should be set to false.

### Reproduction of the problem

http://demos.telerik.com/kendo-ui/treeview/index

Inspect the HTML of the nodes.

### Current behavior
Only when an item is unselected, the aria-selected is set to false.

### Expected/desired behavior
All the non-selected items should have the aria-selected set to false.

### Environment

* **Kendo UI version:** 2017.2.504
* **jQuery version:** 1.12.3
* **Browser:** [all ]

**Comment by kdikov82** [comment]
There are cases when the aria-selected attribute is included initially to the items, but the initial value is incorrect (containing an empty space at the end): aria-selected="false "
http://dojo.telerik.com/uJoni

**Comment by nstoychev** [comment]
Thanks @kdikov82, we will fix this too.
